### ISSUE TITLE
Subroutine arguments with more than one line

### ISSUE THREAD

**Issue by pedro-ricardo** [krvajal/vscode-fortran-support#70]
The subroutine and function dummy-arguments are not highlighted if inserted the like break symbol `&`.
A code like this:
```
subroutine something_large(igxw,igyw,igzw,nxw,nyw,nzw,&
                           igxaux,igyaux,igzaux,nxaux,&
                           nyaux,nzaux,sx,sy,sz,layers,a,b,ngc_aux)
    implicit none
        
        
end subroutine  something_large
```
Is highlighted like this:
![sub_args](https://user-images.githubusercontent.com/15893711/40616190-7bce4abe-6260-11e8-8be9-4eaae558ee34.png)

The highlight fix needs to be done for functions as well.

**Comment by pedro-ricardo** [comment]
Cant tell yet, but I'll look at it this weekend

**Comment by pedro-ricardo** [comment]
The error used to happen in older versions as well. I'm trying to re-write the end part of the rule that highlights the dummy-arguments to look not only for '\n' or ';'. No success yet.

**Comment by pedro-ricardo** [comment]
Still struggling with this regex, will try to finish it by tomorrow, to join it to the release...
Already works but only if the last character in the line above is `&`

**Comment by pedro-ricardo** [comment]
Fixed in #76, to behave just like the string markers, i.e., once opened keep highlighting until a `)` is found.

**Comment by pedro-ricardo** [comment]
Fixed in #76 without drawbacks.
